quickwit/quickwit-metastore/src/tests.rs

```diff
@@ -2261,26 +2260,6 @@ pub mod test_suite {
             split_meta.publish_timestamp,
             Some(split_meta.update_timestamp)
         );
-
-        // wait a sec & re-publish and check publish_timestamp has not changed
```

Comment:
This is no longer possible. Note that this test was utterly artificial because the second publish op used a different delta.